cli/tests/program.rs

```diff
@@ -82,8 +82,17 @@ fn test_cli_program_deploy_non_upgradeable() {
     let mint_keypair = Keypair::new();
     let mint_pubkey = mint_keypair.pubkey();
     let faucet_addr = run_local_faucet(mint_keypair, None);
-    let test_validator =
-        TestValidator::with_no_fees(mint_pubkey, Some(faucet_addr), SocketAddrSpace::Unspecified);
+    let test_validator = TestValidatorGenesis::default()
```

Comment:
Maybe it's worth adding a wrapper function, as this block of code is being repeated in multiple tests.